core/src/main/java/jenkins/views/JenkinsHeader.java

```diff
@@ -0,0 +1,15 @@
+package jenkins.views;
+
+/**
+ * Default {@link Header} provided by Jenkins
+ * 
+ * @see Header
+ */
+public class JenkinsHeader extends FullHeader {
```

Comment:
```suggestion
@Extension(ordinal = Integer.MIN_VALUE)
public class JenkinsHeader extends FullHeader {
```

So we are not creating a new `JenkinsHeader` on every call if the Header is not overridden.